README.md

```diff
@@ -4,39 +4,53 @@ This repository generates the documentation available at https://docs.mattermost
+
+
+**Editing** 
+
+The quickest way to begin is editing directly on GitHub, on your fork of the Mattermost docs repo. 
+
+Click the **Edit** icon on the top right corner of the page you want to edit in the Mattermost documentation.
+
+If this the first time you're contributing, click **Edit the file in your fork of this project** (pencil icon) on the top right corner to create a fork. 
+
+## Creating Pull Requests
+
+1. When you're ready to submit your changes, add a descriptive title and comments to summarize the changes made.
+2. Select **Create a new branch for this commit and start a pull request**.
+3. Check the **Propose file change** button.
+4. Scroll down to compare changes with the original document.
+5. Select **Create pull request**. 
 
-## Contributing
 
-**How to create Pull Requests to edit or create Mattermost documentation (by community members):**
+## Building and Validating
 
-1. Sign the Contributor License Agreement (see instructions in the next section).
-3. On the Mattermost Documentation page that you want to edit, click the GitHub icon on the upper right corner that says "Edit".
-4. Click "Edit the file in your fork of this project" (pencil icon) on the upper right corner.
-5. After making changes, check the "Propose file change" button.
-6. Compare changes with the original document.
-7. Click "Create a Pull Request". Make sure that the Pull Request has a descriptive title and add comments to briefly tell others what you have worked on (optional).
+If you're editing Mattermost documentation on a local machine, you can generate the HTML files from markdown in the `/source` directory to review them before you commit changes or create pull requests. 
 
-**Signing CLA:**
+1. Download the Mattermost docs repo onto a machine with Python installed.
+2. Install [pipenv](https://docs.pipenv.org/): `pip install pipenv`. 
+3. `cd` into the cloned repository.
+4. Install the required packages: `pipenv install`.
+5. Build: `make html` - generates files in `/build` directory.
 
-Please read the [Mattermost Contributor Agreement](https://www.mattermost.org/mattermost-contributor-agreement/) and sign it (at the bottom of the page), so you can be added to the Mattermost [Approved Contributor List](https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true).
 
-**Additional process for Core Committers for Doc Repo:**
+**Additional Process for Core Committers**
 
 These steps are to be followed only in situations of urgency or in situations where it is necessary to check that there aren't any high confidentiality issues. Otherwise the standard approval process should be followed.
```

Comment:
This paragraph should be removed. I don't know why it was originally added or if it's in a wrong place, but the steps described below should be completed as normal day-to-day routine tasks.

Reply:
Thanks Amy, that's what I thought and wanted to make sure. I'll update.